{"name": "Verify User is able to Login with Valid Credentials", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <tests.home.test_login.TestLogin testMethod=test_validLogin>\n\n    @allure.story('HCMAT-12345')\n    @allure.title(\"Verify User is able to Login with Valid Credentials\")\n    @allure.tag('HCMAT-12345')\n    @allure.description(\"Verify User is able to Login with Valid Credentials\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    @pytest.mark.run(order=1)\n    def test_validLogin(self):\n        self.log.info(\"test_validLogin Test is started\")\n        self.ll.login(\"test@email.com\",\"abcabc\")\n        result=self.ll.verifyLoginSuccessfull()\n>       self.ts.markFinal(\"test_validLogin\",result,\"User Able to Login\")\n\ntests\\home\\test_login.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <utilities.teststatus.TestStatus object at 0x042664C0>, testName = 'test_validLogin', result = False, resultMessage = 'User Able to Login'\n\n    def markFinal(self, testName, result, resultMessage):\n        \"\"\"\n        Mark the final result of the verification point in a test case\n        This needs to be called at least once in a test case\n        This should be final test status of the test case\n        \"\"\"\n        self.setResult(result, resultMessage)\n    \n        if \"FAIL\" in self.resultList:\n            self.log.error(testName +  \" ### TEST FAILED\")\n            self.resultList.clear()\n>           assert True == False\nE           AssertionError\n\nutilities\\teststatus.py:63: AssertionError"}, "description": "Verify User is able to Login with Valid Credentials", "attachments": [{"name": " ", "source": "ad3aad5c-abf5-4ada-a1c4-cde577b624c1-attachment.png", "type": "image/png"}, {"name": "log", "source": "0a42c3bf-641e-4b52-a6be-440ecc6c8b57-attachment.txt", "type": "text/plain"}], "start": 1589021970441, "stop": 1589022003761, "uuid": "7c1252ee-1ea8-4049-9354-91ee4a5e45c4", "historyId": "d8b83b6b330a63c153d70aeff9df9e22", "testCaseId": "efdd6fe196cd15e4a46e914b28bd9988", "fullName": "tests.home.test_login.TestLogin#test_validLogin", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "HCMAT-12345"}, {"name": "story", "value": "HCMAT-12345"}, {"name": "parentSuite", "value": "tests.home"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "del1-lhp-N01966"}, {"name": "thread", "value": "36108-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.home.test_login"}]}